libcloudforensics/providers/aws/internal/account.py

```diff
@@ -430,7 +430,7 @@ def CreateVolumeFromSnapshot(
         'AvailabilityZone': snapshot.availability_zone,
         'SnapshotId': snapshot.snapshot_id,
         'TagSpecifications':
-            [common.GetTagForResourceType('volume', volume_name)]
+            [common.CreateTags(common.VOLUME, {'Name': volume_name})]
```

Comment:
It looks to me that you can add multiple tags during EC2 instance creation, if this is the case wouldn't it make sense to add an optional tags(dict[str,str]) parameter to these functions:
CreateVolumeFromSnapshot, GetOrCreateAnalysisVm, Snapshot

you can make it optional and default to {'Name':name} only if None. This can come to handy if an analyst wants to tag the instance with additional information (ex: ticket number), Turbinia does that while starting a VM in GCP, there it relies on add_label after creating the vm, however I will add labels parameter to all the create_ methods in gcp if you add it here for consistency and to have it done in one step instead of two, the api insert request takes labels dict while creation there as well. If you think this is a good idea, open an issue for GCP and assign it to me.